### PR TITLE
solution was added

### PR DESCRIPTION
**Pull request by Maksym845**

### CODE REVIEW

File: src/averageAges.js [hunk 1]
```diff
@@ -14,12 +13,24 @@
...
+    return prev + person;
+  }, 0) / agesArr.length;
+
+  return avg;
+}
+
+function calculateMenAverageAge(people, century = false) {
+  let mens = people.filter(person => person.sex === 'm');
+
+  mens = century !== false
```
Comment (Yeliseiev) [inline src/averageAges.js:29]: I think you don`t need to check is century !== false and you don`t need to equalize it to 'false' in the function declaring

**Review by Yeliseiev — APPROVED** [review]
Good work!

File: src/averageAges.js (hunk 1 above)
Comment (silchyn) [inline src/averageAges.js:29]:
```suggestion
  mens = century
```

**Review by silchyn — CHANGES_REQUESTED** [review]

File: src/averageAges.js [hunk 2]
```diff
@@ -14,12 +13,24 @@
...
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+
+function calculateAvarageAge(people) {
+  const agesArr = people.map(person => person.died - person.born);
+  const avg = agesArr.reduce((prev, person) => {
+    return prev + person;
+  }, 0) / agesArr.length;
+
+  return avg;
```
Comment (silchyn) [inline src/averageAges.js:23]:
```suggestion
  const ages = people.map(person => person.died - person.born);
  
  return agesArr.reduce((prev, person) => {
    return prev + person;
  }, 0) / agesArr.length;
```

**Review by silchyn — CHANGES_REQUESTED** [review]
Good job

File: src/averageAges.js [hunk 3]
```diff
@@ -14,12 +13,24 @@
...
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+
+function calculateAvarageAge(people) {
+  const ages = people.map(person => person.died - person.born);
+  const agesArr = ages.reduce((prev, person) => {
+    return prev + person;
+  }, 0) / ages.length;
```
Comment (naziamoff) [inline src/averageAges.js:21]: You may do the return statement right away. And pay attention to the naming, it should perfectly describe what is stored inside
```suggestion
  const averageAge = ages.reduce((prev, person) => prev + person, 0) / ages.length;
```

**Review by naziamoff — CHANGES_REQUESTED** [review]

**Review by naziamoff — CHANGES_REQUESTED** [review]
I think you should pay attention to use proper names for variables. 
Those mistakes more about English, but I highly recommend to fix them
- men instead of mens
- women instead of womens
- children instead of childrens
- average instead of avarage

**Review by naziamoff — APPROVED** [review]
Nice 🔥

**Pull request closed without merging**